ui/app/src/components/CreateFolderDialog/CreateFolderContainer.tsx

```diff
@@ -64,6 +64,12 @@ export function CreateFolderContainer(props: CreateFolderContainerProps) {
     : itemLookupTable[newFolderPath] !== UNDEFINED;
   const isValid = !isBlank(name) && !folderExists && (!rename || name !== value);
 
+  // If folder name changes, useDetailedItem will try to re-fetch a non-existing item (old path), so we will only re-fetch
+  // if path changes
```

Comment:
This comment is a bit confusing. If the folder name changes the path changes so whether it is useDetailed or this effect, they would both react to that no? What was the idea here? The comment needs to state 1. not to use useDetailedItem and why 2. why/how the effect solves.